ext/nnstreamer/tensor_decoder/tensordecutil.c

```diff
@@ -116,3 +116,4 @@ _free_labels (imglabel_t * data)
   data->total_labels = 0;
   data->max_word_length = 0;
 }
+
```

Comment:
??? Remove this file change from the PR.

Reply:
Fixed.